app/const.go

```diff
@@ -2,8 +2,9 @@ package xpla
 
 // nolint: gosec
 const (
-	appName     = "Xpla"
-	upgradeName = "Genesis"
+	appName              = "Xpla"
+	upgradeName          = "Genesis"
```

Comment:
redundant I think, isn't it?

Reply:
https://github.com/xpladev/xpla/pull/18/commits/139baacbf32f8393ab09c425b7e5ba71a1a1eea5